### PR TITLE
Add carriage return to store information

### PR DESCRIPTION
**Pull request by AntoineMille**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Add carriage return to email address. |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1150 |
| How to test? | Change the size of the company's email with a different size |

### CODE REVIEW

File: themes/classic/modules/ps_contactinfo/ps_contactinfo-rich.tpl [hunk 1]
```diff
@@ -56,7 +56,10 @@
       <div class="data email">
         {l s='Email us:' d='Shop.Theme'}<br/>
        </div>
-       <a href="mailto:{$contact_infos.email}">{$contact_infos.email}</a>
+        {if strlen($contact_infos.email) > 25}
```
Comment (thierrymarianne) [inline themes/classic/modules/ps_contactinfo/ps_contactinfo-rich.tpl]: Where does 25 come from?

**Comment by AntoineMille** [comment]
Normally, it's good.

**Comment by basma-yangui-prestashop** [comment]
@AntoineMille I see that, with your PR, when we make the window smaller, the company's email is not resized but truncated and a part of it is hidden.

ps_6597

I think that, at least, the email should be displayed while mouseover. What do you think @vincentbz ?

**Comment by aleeks** [comment]
@basma-yangui-prestashop can you check again please ?

**Comment by vincentbz** [comment]
Seen with @aleeks , this behaviour is perfect. When it's too long, it goes to the line without hiding any character. Thanks !

**Comment by basma-yangui-prestashop** [comment]
OK. Thank you @aleeks !

**Comment by aleeks** [comment]
Thank you @AntoineMille

**Pull request merged**